src/testdrive/src/action/sql.rs

```diff
@@ -463,8 +468,18 @@ async fn try_run_fail_sql(
     }
 }
 
-pub fn print_query(query: &str) {
-    println!("> {}", query);
+pub fn print_query(query: &str, stmt: Option<&Statement<Raw>>) {
+    use Statement::*;
+    if let Some(stmt) = stmt {
```

Comment:
Should be able to do this all in one `match`:

```suggestion
    if let Some(Statement::CreateSecret(_)) = stmt {
```